tools/clang/test/taef_exec/lit.cfg

```diff
@@ -0,0 +1,40 @@
+# -*- Python -*-
+
+# Configuration file for the 'lit' test runner.
+
+import os
+import platform
+
+import lit.formats
+import lit.util
+
+# name: The name of this test suite.
+config.name = 'clang-taef-exec'
+
+# Check that the object root is known.
+te = getattr(config, 'te', None)
+if te is None:
+    # Otherwise, we haven't loaded the site specific configuration (the user is
+    # probably trying to run on a test file directly, and either the site
+    # configuration hasn't been created by the build system, or we are in an
+    # out-of-tree build situation).
+
+    # Check for 'clang_unit_site_config' user parameter, and use that if available.
```

Comment:
Comment doesn't match code: `'clang_unit_site_config'` should be `'clang_taef_exec_site_config'`.

Reply:
Fixed.